primer.md

```diff
@@ -140,6 +140,7 @@ The following are considered beyond the scope of the specification:
 - Selecting a single identity/access control system
 - Inclusion of protocol-level routing information
 - Event persistence processes
+- Vendor specific Authorisation principles
```

Comment:
I think we should rat-hole on "Authorisation" vs "Authorization"  :-)

Reply:
*Authorization* it is!